packages/bundle/src/adaptiveCards/Attachment/CommonCard.js

```diff
@@ -1,42 +1,29 @@
 import { connectToWebChat } from 'botframework-webchat-component';
 import AdaptiveCardBuilder from './AdaptiveCardBuilder';
 import AdaptiveCardRenderer from './AdaptiveCardRenderer';
-import memoize from 'memoize-one';
-import PropTypes from 'prop-types';
-import React from 'react';
 
-class CommonCard extends React.Component {
-  constructor(props) {
-    super(props);
+import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
 
-    this.buildCard = memoize((adaptiveCards, content, styleOptions) => {
-      const builder = new AdaptiveCardBuilder(adaptiveCards, styleOptions);
+const CommonCard = ({ adaptiveCardHostConfig, adaptiveCards, attachment: { content }, styleSet: { options } }) => {
+  const buildCard = useMemo(() => {
```

Comment:
I think this is a noun. Something like `cardBuilt`. It should convey the idea that "the card built using JSON and the Adaptive Card renderer".

And please fix others.